### PR TITLE
Frontend modification and unit testing on footer 

### PR DESCRIPTION
**Pull request by yywu123**
Do the change in the nav bar.
And learn how to do a unit testing on footer

### CODE REVIEW

File: client/src/__tests__/Footer.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    });
+    
+    test('should render Footer component', () => {
+        render(<Footer/>);
+        const EmailElement = screen.getByRole("link", 'mailto:email@domain.com');
+        expect(EmailElement).toBeInTheDocument();  
+        expect(EmailElement).toBeEnabled();  
+
+        const imageElement = screen.getByRole('img',{class: /contact/i})
+        expect(imageElement).toBeInTheDocument();
```
Comment (bhusalanil121) [inline client/src/__tests__/Footer.test.js:19]: Good Work at unit Testing

**Review by bhusalanil121 — COMMENTED** [review]
Good detailed work on NavBar. I would recommend to use different color effect to see where the user is  ..... For eg: If user is at profile page ... lets indicate that  with some color or something that make sense that user is at that specific page.

**Comment by yywu123** [comment]
> 
Good recommendation. I may look into it if I have time.

File: client/src/__tests__/Footer.test.js (hunk 1 above)
Comment (yywu123) [reply client/src/__tests__/Footer.test.js:19]: Good recommendation. I may look into it if I have time.

**Comment by alicechen1998** [comment]
dont merge yet I want to check it out tomorrow night, please

**Comment by bhusalanil121** [comment]
Please solve merge conflict and merge it asap.

**Comment by tenzin1308** [comment]
@yywu123 great work on testing.
could you please fix those minor conflict so we can merge it to main

**Pull request merged**